aria_plugin/operations.py

```diff
@@ -31,13 +31,16 @@
 @operation
 def create(**_):
     env = Environment(ctx)
-    # Make sure no other service was created under the same name
+    # Make sure no other service template was created under the same name
```

Comment:
Make sure there is no other stored service template with the same name.
We check this here, and not catching the exception that ARIA raises in this case since we want to preform this check before any 'heavy-lifting' operations.